backend/sirius-web-graphql/src/main/java/org/eclipse/sirius/web/graphql/datafetchers/editingcontext/EditingContextChildCreationDescriptionsDataFetcher.java

```diff
@@ -24,32 +26,31 @@
 import graphql.schema.DataFetchingEnvironment;
 
 /**
- * Computes the child creation descriptors for a specific class.
+ * The data fetcher used to retrieve the child creation descriptions accessible in an editing context.
  * <p>
  * It will be used to fetch the data for the following GraphQL field:
  * </p>
  *
  * <pre>
- * type Viewer {
- *   childCreationDescriptions(classId: ID!): [ChildCreationDescription!]!
+ * type EditingContext {
+ *   rootObjectCreationDescriptions(namespaceId: ID!, suggested: Boolean!): [ChildCreationDescription!]!
  * }
  * </pre>
  *
- * @author sbegaudeau
+ * @author hmarchadour
```

Comment:
It seems the Javadoc between this and `EditingContextRootObjectCreationDescriptionsDataFetcher` have been inverted.